src/controllers/authController.ts

```diff
@@ -0,0 +1,75 @@
+import { Request, Response } from "express";
+import User from "../models/userModel";
+import bcrypt from "bcrypt";
+import jwt from "jsonwebtoken";
+import { sendConfirmEMail } from "../config/nodemailerConfig";
+import { generateCode } from "../config/generateCode";
+import * as dotenv from 'dotenv'
+
+dotenv.config();
+
+export const createUser = async (req: Request, res: Response) => {
```

Comment:
This function is doing too much...Limit it to just creating user and returning response